docs/usage/mend-hosted/credentials.md

```diff
@@ -111,7 +111,14 @@ Calls to the APIs require authentication with a personal access token from the G
 The token must be included in the request header as the `authorization` property.
 
 You can use a [classic token](https://github.com/settings/tokens) or a [fine-grained token](https://github.com/settings/personal-access-tokens).
-No additional scopes or permissions are required for the token.
+Mend recommends the use of fine-grained tokens for enhanced security.
+
+#### Scopes and permissions for tokens
+
+When calling APIs on _private_ repositories, the following scopes and permissions are required:
```

Comment:
```suggestion
To be able to update secrets on both public and private repositories, the following scopes and permissions are required:
```

Reply:
In fact I think you should consider simplifying it further by describing the one, universal way to get all scenarios working with fine-grained tokens.